src/vs/workbench/contrib/welcomeViews/common/newFile.contribution.ts

```diff
@@ -32,11 +32,13 @@ registerAction2(class extends Action2 {
 				primary: KeyMod.Alt + KeyMod.CtrlCmd + KeyMod.WinCtrl + KeyCode.KeyN,
 				weight: KeybindingWeight.WorkbenchContrib,
 			},
-			menu: {
-				id: MenuId.MenubarFileMenu,
-				group: '1_new',
-				order: 2
-			}
+			// {{SQL CARBON EDIT}} - Start
```

Comment:
SQL CARBON EDITs should always have text explaining why we're making a change - in this case `We have our own New File item` should be enough

But did you look to see if we can just use theirs and get rid of ours? If they have an item to make a new blank file already I don't see the reason to keep ours around.

Reply:
Updated.

Yes I took a look at theirs and it's not very straightforward to just use this and clean up `New File` and `New Query`. Given that I don't have time to look deep into this now, I'll just comment out this part.